### DISCUSSION TITLE
hospitalization data

### DISCUSSION THREAD

**Discussion by Elkabeielika** [owid/covid-19-data#2671] [Q&A] [answered]
How to calculate weekly admissions for every single week when in all days of week have a number? By sum or avarege function? Thanks

**Comment by lucasrodes** [comment, marked as answer]
Hi @Elkabeielika,
Thanks for raising this.

Weekly metrics are obtained by summing the metric values on the reporting date and the 6 preceding days.

**Reply by lucasrodes** [reply]
I tried to improve the documentation in 20607b91e9568b16ffed3355272029df629ea65d

**Reply by naczek** [reply]
Hi, 
What if I need to calculate the cumulative number of hospitalized people? Should I sum up every 7th value?
Thank you
